README.md

```diff
@@ -149,50 +148,53 @@ The first round airdrop will start at around `2023-03-15-06:00 UTC`, and the sec
 }
 ```
 
+The airdrop has two rounds: In the first round, all participants will go through the same flow, and the airdrop will start at around `2023-03-15 06:00 UTC`. In the second round, all participants will go through different flows, and the airdrop will start at around `2023-03-16 06:00 UTC`. Before that, participants can prepare their strategies. Please ensure that you have been added to the gon-evidence repository and that your information is correct, otherwise you may not receive the airdrop!
+
 **Rules:**
-- The `flow` can be queryied with `flow-id` [here](https://github.com/game-of-nfts/gon-testnets/blob/main/doc/flow-table.md). 
-- The `start_height` is the block height on IRISnet. Any transfer prior to the `start_height` will be considered invalid.
-- Participants must use their own account addresses provided at the registration stage.
-- The `last_owner` is an official address on IRISnet. After finishing the interchain NFT transfer, every participant transfer the NFT to `last_owner` then complete the race.
-- Considering the time zone difference, the completion time is calculated by the difference in height between the last NFT transfer (to the `last_owner`) and the first interchain NFT transfer on IRISnet.
+
+- The `start_height` is the block height on IRISnet. Any transfer prior to the `start_height` will be considered invalid, which means your first Interchain NFT Transfer must happen after that height.
```

Comment:
Since we don't know what the start_height will be, will we have enough time after the airdrop to actually prepare or will the start_height be so soon after the airdrop that this needs to be scripted?
Especially considering time-zone differences.

Reply:
Or does start_time simply mean "start after block"? As long as you start after block it is fine? I am asking because of the time-difference comment below.

I guess the confusion is because this is considered a race, and start_height sounds like when we should start :D

Reply:
The plan is to set `start_height` as the height around the aridrop time. Considering the time zone we use the block height to  replace the real time. Note `start_height` only means your first interchain transfer tx will be considered valid after that height, the real first transfer time is up to you.

```bash 
start_height = 10000

first_interchain_transfer = 13000

last_transfer_to_last_owner = 13500

your difference of height is 13500-13000 == 500, and that's your grade

end_height = 20000 (suppose it's 20000; in fact, it will be the block height of IRIS at 03-17-23:59:59)
```

Invalid exmaple 1:
```
first_interchain_transfer = 9000  (you jumped the gun, you won't get points from B1~B4)
start_height = 10000
```

Invalid exmaple 2:
```
start_height = 10000

first_interchain_transfer = 13000

end_height = 20000

last_transfer_to_last_owner = 20001 ( we don't rank you, but you still get points from B1 and B2)
```

Reply:
> Or does start_time simply mean "start after block"? As long as you start after block it is fine? I am asking because of the time-difference comment below.
> 
> I guess the confusion is because this is considered a race, and start_height sounds like when we should start :D

Do you have any suggestion on its(`start_height`) naming?

Reply:
What about `start_after_height`?